android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayout.java

```diff
@@ -284,6 +313,52 @@ public void didSelectTab(Tab tab, @TabSelectionType int type, int lastId) {
     };
   }
 
+  private void addSavedBandwidthToDb(long savings) {
```

Comment:
this is ridiculously overcomplicating things. The value is already stored in a pref that we can read directly in java. Storing this stuff in a database is completely unnecessary